### ISSUE TITLE
Error while Team Collaboration

### ISSUE THREAD

**Issue by abdulrahman-fullstack** [codesandbox/codesandbox-client#4321]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 80.0.3987 | Linux x86_64           | 550c67259          |

  **Route:**
  https://codesandbox.io/s/react-sandbox-team-laby-rwsdu?file=/src/components/Suhail/Home.js

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'username' of undefined
  TypeError: Cannot read property 'username' of undefined
    at https://codesandbox.io/static/js/app.2ba592bba.js:571:1021779
    at Array.map (<anonymous>)
    at Cu (https://codesandbox.io/static/js/app.2ba592bba.js:571:1021301)
    at ua (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1733563)
    at Ka (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1741828)
    at Es (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1780016)
    at ml (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1764402)
    at hl (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1764327)
    at il (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1761655)
    at https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719838
    at t.unstable_runWithPriority (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1795174)
    at qo (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719547)
    at Yo (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719783)
    at $o (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719718)
    at el (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1758441)
    at Sa (https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1737236)
    at forceRerender (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:922189)
    at e.callback (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:1043031)
    at b.flush (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:1045951)
    at h.flush (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:1042085)
    at Object.cb (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:931462)
    at e.emit (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:925307)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:933869
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935541)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928723
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:929390)
    at a (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:929467)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935900
    at Object.o [as operator] (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:929200)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928685)
    at s (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928854)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935999
    at o (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928454)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935541)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928777)
    at s (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928854)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935757
    at o (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928454)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935541)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928777)
    at s (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928854)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935757
    at o (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:928454)
    at i (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935541)
    at t (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:935578)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:933737
    at new Promise (<anonymous>)
    at live.liveMessageReceived (https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:933717)
    at https://codesandbox.io/static/js/vendors~app.3f6ad0ce6.chunk.js:1:933027
    at e.channel.channel.onMessage (https://codesandbox.io/static/js/app.2ba592bba.js:571:386464)
  
    in Cu
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in c
    in t
  ```

  </details>

**Comment by garethx** [comment]
@abdulrahman-fullstack What were you trying to accomplish when the crash occurred?

**Comment by CompuIves** [comment]
Ah, I think #4176 is related to this.

**Comment by lbogdan** [comment]
Closing this, as it should be fixed by #4167 .
